controllers/client.controller.js

```diff
@@ -0,0 +1,37 @@
+var clients = require('../models/clients');
```

Comment:
This controller is working with two resources, the invoices and projects, yet it is called clientController.  Generally your controller should match the resource (invoiceController, projectController).